src/scripts/git_data.py

```diff
@@ -13,6 +17,36 @@ def get_last_commit_date(file_path):
         print(f"An error occurred: {e}")
     return None
 
+def get_last_commit_date(file_path):
+    try:
+        # get the last commit date as "September 12, 2022"
+        GITHUB_TOKEN = os.environ['GITHUB_TOKEN']
+
+        if GITHUB_TOKEN is None:
+            print('Github token not found')
+            return get_last_commit_date_by_terminal_log(file_path)
+        
+        headers = {
+            'X-GitHub-Api-Version': '2022-11-28',
```

Comment:
is this really needed?

Reply:
The call works without it but the documentation had specified it. https://docs.github.com/en/rest/using-the-rest-api/getting-started-with-the-rest-api?apiVersion=2022-11-28#4-make-a-request-with-github-cli

Would you like to remove it?

Reply:
That's fine, thanks for clarifying!